app/src/main/kotlin/cz/covid19cz/erouska/jobs/AutoRestartJob.kt

```diff
@@ -0,0 +1,84 @@
+package cz.covid19cz.erouska.jobs
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Handler
+import cz.covid19cz.erouska.BuildConfig
+import cz.covid19cz.erouska.service.CovidService
+import cz.covid19cz.erouska.utils.L
+import org.threeten.bp.ZonedDateTime
+
+class AutoRestartJob : BroadcastReceiver() {
+
+    companion object {
+        const val EXTRAKEY_START = "AUTO_RESTART_START"
+        const val EXTRAKEY_CANCEL = "AUTO_RESTART_CANCEL"
+
+        private const val ACTION = "EROUSKA_ALARM"
```

Comment:
asi bych tu action trosku vic specifikoval..btw skoro bych asi pouzival jenom ty actions and URI bych neresil. ale je to vicemene jedno…jen to takhle moc nevidam.

Reply:
Hele já jsem právě myslel, že akce bude obecná, URI scheme taky a URI path bude říkat, co se má udělat. Ale nemám problém to rozlišit jménem akce.

Reply:
A nebo intent extra. Ale jak rikam, je to vicemene fuk..

Reply:
Poladil jsem to na `EROUSKA_RESTART` a `erouska://restart`.